src/platform.rs

```diff
@@ -163,6 +182,11 @@ impl Platform {
             // No NEON compress_xof() implementation yet.
             #[cfg(blake3_neon)]
             Platform::NEON => portable::compress_xof(cv, block, block_len, counter, flags),
+            #[cfg(blake3_wasm32_simd)]
+            // TODO Safe because compiled for wasm32
```

Comment:
Similar to my comments around the intrinsics: Hopefully we can just make this safe to call? If so the comment should probably be something like "this will crash if the host doesn't support Wasm SIMD".

Reply:
The only safety concern now is correct stores and loads.  I will think about formulating the comment.

Reply:
I've made some functions safe with small unsafe blocks inside.  I haven't converted `hash_many` and related functions, though.